package.json

```diff
@@ -27,17 +28,30 @@
   },
   "homepage": "https://github.com/makeomatic/fingerprint#readme",
   "dependencies": {
-    "bluebird": "^3.0.6",
+    "@types/callsite": "1.0.30",
+    "@types/isstream": "0.1.0",
+    "@types/node": "13.13.1",
```

Comment:
I can move them to devDependencies, but the result will be that these type dependencies will not be downloaded when this library is used as a dependency. This will limit effectiveness of intellisense because some of the types might not be available to the editor. Let me know if you'd still like me to move them to devDependencies

Reply:
please do, its not supposed to be installed for a production build